website/content/partials/versions.mdx

```diff
@@ -0,0 +1,19 @@
+## Vault Version Guidance
+
+HashiCorp regularly releases new versions of Vault in the form of "major" and "minor" releases.
+Vault uses version numbers in the form of X.Y.Z.
```

Comment:
We've occasionally (not recently) had ent versions of the form X.Y.Z.A, should we mention that for completeness?